source/scripts/add.js

```diff
@@ -214,6 +214,17 @@ function saveTaskToStorage(tasks) {
   localStorage.setItem("tasks", JSON.stringify(tasks));
 }
 
+/**
+ * Customized error class to handle invalid input when the form is submitted
+ * Invalid input corresponds to negative minutes/hours or minutes greater than 60
+ */
+class invalidTime extends Error{
```

Comment:
I would capitalize class name per coding convention but we never explicitly wrote it in our coding protocol so I will leave the decision up to you.